src/chainrules.jl

```diff
@@ -6,3 +6,122 @@ ChainRulesCore.@scalar_rule(
     ),
     (x, - tanh(Ω + b) * x, x - 1),
 )
+
+# `OrderedBijector`
+function ChainRulesCore.rrule(::typeof(_transform_ordered), y::AbstractVector)
```

Comment:
All these rules should be tested with ChainRulesTestUtils (as done for other rules).

Reply:
Where is the other rules we test? Just need something to go on.

Reply:
In ad/chainrules.jl IIRC.